src/main/java/com/jcabi/ssh/SSH.java

```diff
@@ -182,6 +193,17 @@ public SSH(final String adr, final int prt,
     }
 
     /**
+     * Constructor using SSHOptions object.
+     * @param options SSHOptions options object with connection properties
+     * @throws UnknownHostException If fails
```

Comment:
@LinkedList the exception is quite specific, but the description is generic. Let's make the description more specific